tensorboard/webapp/widgets/range_input/range_input_component.ts

```diff
@@ -245,10 +250,14 @@ export class RangeInputComponent implements OnInit, OnDestroy {
       xPositionInPercent = compensatedRelativeXInPx / (right - left);
     }
 
-    const newValue = this.getClippedValue(
+    let newValue = this.getClippedValue(
       this.min + (this.max - this.min) * xPositionInPercent
     );
 
+    if (this.returnIntegers) {
+      newValue = Math.round(newValue);
```

Comment:
Since we have just rounded the value there should be no concern around floating point math so could we just return here?
```suggestion
      return Math.round(newValue);
```

Reply:
Yea that makes sense. Then I can change that back to a const.